MvvmCross-Forms/MvvmCross.Forms/Presenters/MvxFormsPagePresenter.cs

```diff
@@ -20,32 +20,20 @@ public abstract class MvxFormsPagePresenter
     {
         public const string ModalPresentationParameter = "modal";
 
-        private Application _mvxFormsApp;
-
-        public Application MvxFormsApp
+        private MvxFormsApplication _formsApplication;
+        public MvxFormsApplication FormsApplication
```

Comment:
Yes. But it is necessary because otherwise we can't extend this with MvvmCross methods.